packages/scheduled/src/index.ts

```diff
@@ -112,7 +112,7 @@ export const throttle: ScheduleCallback = (callback, wait) => {
 export const scheduleIdle: ScheduleCallback = isServer
   ? () => Object.assign(() => void 0, { clear: () => void 0 })
   : // requestIdleCallback is not supported in Safari
-    (window.requestIdleCallback as typeof window.requestIdleCallback | undefined)
+    (typeof window !== 'undefined' && window.requestIdleCallback as typeof window.requestIdleCallback | undefined)
```

Comment:
```suggestion
    (typeof requestIdleCallback !== 'undefined')
```
Actually, wouldn't this be better?
since it's on a global scope anyway
if it's present, use it, that's the point
`window` is unnecessary